src/NUnitFramework/framework/Internal/AssemblyHelper.cs

```diff
@@ -92,7 +92,7 @@ public static AssemblyName GetAssemblyName(Assembly assembly)
 
         #region Load
 
-#if NETSTANDARD1_6
+#if NETSTANDARD1_6 || NETSTANDARD2_0
```

Comment:
I like to try to call out the relevant platforms. This only works on .NET Core but the closest I can get is `NETSTANDARD1_6 || NETSTANDARD2_0`. .NET Framework is not relevant here in the same way that if we had a UWP target it would also not be relevant here. At least that's the angle I saw. Do you see a different angle?

Reply:
No. I'm just a bit confused 😄 .

Reply:
Okay. If for example NUnit had both .NET Standard and .NET Core targets, I'd have used `#if NETCOREAPP1_1 || NETCOREAPP2_0` instead because that section only works on .NET Core.

Reply:
And then I also wouldn't need to use reflection 😆